### PR TITLE
[CM-1695] Disable Select/Deselect of check box in new UI once form is submitted

### PR DESCRIPTION
**Pull request by AbhijeetRanjan308**
## Summary
-  Created a configuration for Form to Disable after Submission. 
```
Kommunicate.defaultConfiguration.disableFormPostSubmit = true
```

## Working

1. If the configuration is enabled, the form will be disabled after the form submission.
2. The form will also be disabled if the user sends a message (the person filling the form).
3. The form will not be disabled if the client sends a message after sending form (the person who send the form to user). The form will only be disabled in the case of the form submission or if the user sends a message.
4. The color of the form changes when the form is submitted. In this configuration, `KMMultipleSelectionConfiguration.shared.enableMultipleSelectionOnCheckbox = true`, the color of the text, border, and checkbox will change to darkGray. When the KMMultipleSelection configuration is disabled, the color of the tick changes to darkGray in both single and multiple selections.

## Testing

- [x] Tested in SPM

## Video Sample

Form with `KMMultipleSelectionConfiguration.shared.enableMultipleSelectionOnCheckbox = true`

https://github.com/Kommunicate-io/KommunicateChatUI-iOS-SDK/assets/139110221/675456b4-0d4e-4903-adf2-ad4c85d583a3

Normal From

https://github.com/Kommunicate-io/KommunicateChatUI-iOS-SDK/assets/139110221/00a98c1c-08fa-4760-add6-bdf020b7385a

SPM Testing Video

https://github.com/Kommunicate-io/KommunicateChatUI-iOS-SDK/assets/139110221/d0aca32e-f925-4e46-ad12-65be5e0dc607

### CODE REVIEW

File: Sources/Utilities/KMMultipleSelectionConfiguration.swift [hunk 1]
```diff
@@ -29,8 +31,12 @@ public struct KMMultipleSelectionConfiguration {
     
     public var borderColor: UIColor = UIColor(hexString: "00A4BF")
     
+    public var afterSubmitBorderColor: UIColor = UIColor(hexString: "8F8F8F")
```
Comment (Sathyan-KM) [inline Sources/Utilities/KMMultipleSelectionConfiguration.swift:34]: postsubmitbordercolor

File: Sources/Views/ALKFormCell.swift [hunk 2]
```diff
@@ -276,7 +281,11 @@ extension ALKFormCell: UITableViewDataSource, UITableViewDelegate {
             } else {
                 cell.accessoryType = .none
             }
-
+            if let check = viewModel?.isFormSubmitted() , check {
```
Comment (Sathyan-KM) [inline Sources/Views/ALKFormCell.swift:284]: rename the variable `check`

Comment (AbhijeetRanjan308) [reply Sources/Views/ALKFormCell.swift:284]: Changed the name to checkForFormSubmitted

File: Sources/Utilities/KMMultipleSelectionConfiguration.swift (hunk 1 above)
Comment (AbhijeetRanjan308) [reply Sources/Utilities/KMMultipleSelectionConfiguration.swift:34]: done

**Review by Sathyan-KM — APPROVED** [review]

**Pull request merged**